blaze/tests/test_interactive.py

```diff
@@ -94,6 +95,13 @@ def test_repr():
     assert '...' in result
 
 
+def test_str_does_not_repr():
+    # see GH issue #1240.
+    d = Data([('aa', 1), ('b', 2)], dshape='2 * {a: string, b: int64}')
+    expr = transform(d, c=d.a.strlen() + d.b)
+    assert sub(r'_\d+', 'XXX', str(expr)) == "Merge(_child=XXX, children=(XXX, label(strlen(_child=XXX.a) + XXX.b, 'c')))"
```

Comment:
I thought of doing that, but my concern is that the `_149` (I think that's the symbol) is not necessarily reproducible run-to-run, especially with `py.test` running things in parallel.  Is it reproducible?  If so, then I'll make the replacement and remove the `re.sub` call.